test/e2e/common/volumes.go

```diff
@@ -75,7 +75,7 @@ var _ = framework.KubeDescribe("GCP Volumes", func() {
 	////////////////////////////////////////////////////////////////////////
 
 	framework.KubeDescribe("NFSv4", func() {
-		It("should be mountable for NFSv4 [Volume]", func() {
+		It("should be mountable for NFSv4 [sig-storage]", func() {
```

Comment:
I think the problem with moving things out of common is that they will no longer be run by the node e2e test.